src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,43 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    void testSetSize() {
+        int setSize = numbers.size();
+        assertThat(setSize).isEqualTo(3);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1,2,3})
+    void testContains(int number) {
+        assertThat(numbers.contains(number)).isTrue();
+    }
```

Comment:
```suggestion
    void testSetSize() {
        assertThat(numbers).hasSize(3);
    }

    @ParameterizedTest
    @ValueSource(ints = {1,2,3})
    void testContains(int number) {
        assertThat(numbers).contains(number);
    }
```
컬렉션에 있는 값들을 꺼내보지 않고도 테스트할 수 있게 편의 메소드를 제공하고 있습니다. 😄

Reply:
assertThat 내부에 BooleanAssert가 따로 있군요,,
감사합니다